float32 camera_angle
